be/src/runtime/statistic_result_writer.cpp

```diff
@@ -152,6 +153,9 @@ StatusOr<TFetchDataResultPtr> StatisticResultWriter::_process_chunk(Chunk* chunk
     } else if (version == STATISTIC_EXTERNAL_VERSION) {
         RETURN_IF_ERROR_WITH_WARN(_fill_full_statistic_data_external(version, result_columns, chunk, result.get()),
                                   "Fill table statistic data failed");
+    } else if (version == STATISTIC_EXTERNAL_QUERY_VERSION) {
+        RETURN_IF_ERROR_WITH_WARN(_fill_full_statistic_query_external(version, result_columns, chunk, result.get()),
```

Comment:
same with _fill_full_statistic_data_external?

Reply:
The biggest difference is one use HLL when collect statistics and one use int64 when query statistics.